apps-rendering/src/components/Callout/formFields.tsx

```diff
@@ -49,6 +62,8 @@ export const FormField: FC<FormFieldProp> = ({
 			<Option value="">Please select an option</Option>,
 		);
 
+	if (formField.hidden) return null;
```

Comment:
Do we need to render the `<input type="hidden" />` to make sure this works without JS?

Reply:
Yea sounds good, just changed it 👍